app/components/hiring_staff/vacancies_component.html.haml

```diff
@@ -2,22 +2,16 @@
   .govuk-grid-column-one-quarter
     = form_for @filters_form, url: organisation_managed_organisations_path, html: { method: "patch" } do |f|
       = render 'shared/filter_group', f: f,
-        filters: { title: 'Job filters', items: [{ title: 'Locations', attribute: :managed_school_ids, options: @school_options, value_method: :id, text_method: :name, small: true }] }
+        filters: { title: 'Filter results', :removeButtons => true, :totalCount => @filters[:managed_school_ids]&.count, items: [{ title: 'Locations', key: 'locations', search: true, attribute: :managed_school_ids, selected: @filters[:managed_school_ids], options: @school_options, value_method: :id, text_method: :name, small: true }] }
 
       = f.hidden_field :jobs_type, value: @selected_type
       = f.govuk_submit t('buttons.apply_filters')
 
 %div{ class: grid_column_class }
-  .dashboard-header
-    - if @organisation.is_a?(SchoolGroup)
-      .filters-information
-        %button.govuk-button.govuk-button--secondary{ class: "govuk-!-margin-bottom-0" }= t('buttons.hide_filters')
```

Comment:
Where has this button moved to?

Reply:
it was removed as it is part of an old figma design

Reply:
Which Figma are you using?